src/views/home/UserInput.js

```diff
@@ -39,55 +47,81 @@ export default class UserInput extends Component {
   onFormSubmit = async e => {
     e.preventDefault();
     const body = JSON.stringify(this.state);
-
-    try {
-      const res = await createTicket(body);
-
+    const res = await createTicket(body);
```

Comment:
Is it OK to move this out of the `try` block? Could something in there throw? I guess we're not showing the user anything right now if it fails anyway...

Reply:
It's currently not in a try catch block. I think I took that whole bit out.